stuff/pipelines/test_course_callback.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python3
+
+'''test course_callback module.'''
+
+from ast import alias
+import dataclasses
+import datetime
+import json
+import os
+import random
+import string
+import sys
+import time
+
+from typing import List
+
+import omegaup.api
+
+import course_callback
+import rabbitmq_connection
+import test_constants
+import test_credentials
+
+
+sys.path.insert(
+    0,
+    os.path.join(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "."))
+import lib.db   # pylint: disable=wrong-import-position
+
+
+def test_insert_course_certificate() -> None:
+    '''Test get course participants'''
+
+    client = omegaup.api.Client(api_token=test_constants.API_TOKEN,
+                                url=test_constants.OMEGAUP_API_ENDPOINT)
+    current_time = datetime.datetime.now()
+    past_time = current_time - datetime.timedelta(hours=5)
+    future_time = current_time + datetime.timedelta(hours=5)
+    course_alias = ''.join(random.choices(string.digits, k=8))
+    assignment_alias = ''.join(random.choices(string.digits, k=8))
+
+    # Creating a course with an assignment and then adding problem and users
+    client.course.create(
+        name=course_alias,
+        alias=course_alias,
+        description='Test course',
+        start_time=time.mktime(current_time.timetuple()),
+        finish_time=time.mktime(future_time.timetuple()),
+        objective='Testing',
+        level='intermediate',
+        show_scoreboard=1,
+        requests_user_information='no',
+    )
+
+    client.course.createAssignment(
+        name=assignment_alias,
+        alias=assignment_alias,
+        course_alias=course_alias,
+        assignment_type='homework',
+        description='Test course assignment',
+        start_time=time.mktime(current_time.timetuple()),
+        finish_time=time.mktime(future_time.timetuple()),
+    )
+
+    assignmentDetails = client.course.assignmentDetails(
+        course=course_alias,
+        assignment=assignment_alias)
+
+    client.course.addProblem(
+        course_alias=course_alias,
+        assignment_alias=assignment_alias,
+        problem_alias='sumas',
+        points=1.0,
+    )
+
+    usernames: List[str] = []
+    for number in range(3):
+        user = f'course_test_user_{number}'
```

Comment:
```suggestion
        username = f'course_test_user_{number}'
```

nit: esto no es un usuario, sino sólo su nombre. el usuario sería el objeto completo en la base de datos.

Reply:
Listo